app/src/main/res/values/strings.xml

```diff
@@ -63,8 +63,12 @@
     <!-- Shared preferences keys -->
     <string name="pref_notifications">pref_notifications</string>
 
-
+    <string name="placeholder_date">July 25, 2018</string>
+    <string name="comments">Comments</string>
+    <string name="ipsum_lorem_paragraph">Lorem ipsum dolor sit amet, harum quando mel ad. No usu graece perfecto interesset, eam vide consul pertinacia eu, suas labores percipit ius ne. No usu libris mucius, quo sonet iudicabit assueverit ei, ea dolore maiestatis quaerendum mea. Mel at congue elaboraret, nam viris voluptatibus eu. Ius dissentiet comprehensam te. Te ancillae petentium salutandi per, est no congue iriure.</string>
```

Comment:
I think this is for the comment body if none is found, use a string like "no content" instead of a long test string.

Reply:
@weihanli101  Im going to approve this now but we should not forget about this.